deutsche-gesellschaft-fur-psychologie.csl

```diff
@@ -121,23 +130,27 @@
   </macro>
   <macro name="publisher">
     <choose>
-      <if type="report thesis" match="any">
-        <group delimiter=": ">
-          <text variable="publisher-place"/>
-          <text variable="publisher"/>
-        </group>
-      </if>
-      <else>
+      <if type="article-newspaper" match="none">
         <choose>
-          <if variable="event" match="none">
-            <text variable="genre" suffix=", "/>
+          <if type="report thesis" match="any">
+            <group delimiter=": ">
+              <text variable="publisher-place"/>
+              <text variable="publisher"/>
+            </group>
           </if>
+          <else>
+            <choose>
+              <if variable="event" match="none">
+                <text variable="genre" suffix=", "/>
+              </if>
+            </choose>
+            <group delimiter=": ">
+              <text variable="publisher-place"/>
+              <text variable="publisher"/>
+            </group>
+          </else>
         </choose>
```

Comment:
@damnation333, this is a little odd, by the way. You always print the publisher-place/publisher information, so it should ideally be moved out of the `<choose/>`. The `type="report thesis"` conditional can then be used to directly control the printing of the "genre" variable, e.g.:

```xml
            <choose>
              <if type="report thesis" variable="event" match="none">
                <text variable="genre" suffix=", "/>
              </if>
            </choose>
```